src/NzbDrone.Core/IndexerSearch/EpisodeSearchService.cs

```diff
@@ -66,6 +66,41 @@ public void MissingEpisodesAiredAfter(DateTime dateTime, IEnumerable<Int32> grab
             _logger.ProgressInfo("Completed search for {0} episodes. {1} reports downloaded.", missing.Count, downloadedCount);
         }
 
+        private void SearchForMissingEpisodes(List<Episode> episodes)
+        {
+            _logger.ProgressInfo("Performing missing search for {0} episodes", episodes.Count);
+            var downloadedCount = 0;
+
+            using (var rateGate = new RateGate(100, TimeSpan.FromSeconds(60)))
```

Comment:
The idea is that we won't spam indexers looking for missing episodes in batch (namely when doing a search for all missing), but yes, the numbers don't match.